core/include/kernel/ts_store.h

```diff
@@ -45,8 +45,8 @@ struct ts_store_ops {
 	 * @data_user: pointer to user memory where the TS bytes should be
 	 *             copied.
 	 * At least one of @data_core and @data_user and normally NULL, but
-	 * both are also permitted to be non-null.
-	 * If @data_core == NULL, @data_user, and @len != 0, the function
+	 * both are also permitted to be non-NULL.
+	 * If @data_core == NULL and @data_user, and @len != 0, the function
```

Comment:
This is still a bit weird, we want both data pointers NULL right? So "If @data_core == NULL and @data_user == NULL and len != 0, the function..."

Reply:
Right, I'll update.